plugins/storage/volume/scaleio/src/main/java/org/apache/cloudstack/storage/datastore/driver/ScaleIOPrimaryDataStoreDriver.java

```diff
@@ -189,6 +189,11 @@ public boolean grantAccess(DataObject dataObject, Host host, DataStore dataStore
 
     @Override
     public void revokeAccess(DataObject dataObject, Host host, DataStore dataStore) {
+        if (host == null) {
+            LOGGER.warn("Declining to revoke access to PowerFlex volume when a host is not provided");
```

Comment:
could be `info`, right? or is this indicative of a possible (user) error?

Reply:
You're right @DaanHoogland - it could probably be info. My thinking was that yes, it could potentially mean that the caller did something wrong, but not necessarily.